rasa_core/policies/ensemble.py

```diff
@@ -350,6 +351,21 @@ def probabilities_using_best_policy(self,
                 result = probabilities
                 best_policy_name = 'policy_{}_{}'.format(i, type(p).__name__)
                 best_policy_priority = p.priority
+                best_policy_index = i
+
+        if logger.level == logging.DEBUG:
+            # When fallback was invoked because of low core_threshold,
+            # log the information so that people know it was the action
+            # prediction and not the intent classification that messed up.
+            fallback_idx_policy = [(i, p) for i, p in enumerate(self.policies)
+                                   if isinstance(p, FallbackPolicy)]
+            if fallback_idx_policy:
+                fallback_idx, fallback_policy = fallback_idx_policy[0]
+                if fallback_idx == best_policy_index and \
+                        fallback_policy.core_threshold_set:
+                    logger.debug("Fallback policy was invoked because of "
+                                 "core_threshold ({})"
```

Comment:
Perhaps something like `"Highest action prediction confidence is lower than core threshold {}.".format(core_threshold)`
(logger will already tell you that fallback policy was used to predict the next action, I think the goal here is more to clarify that there wasn't a confident enough action to get above the threshold)